bsp/boards/radio.h

```diff
@@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
 void     radio_rfOn(void);
 void     radio_rfOff(void);
 // TX
-void     radio_loadPacket(uint8_t* packet, uint8_t len);
+void     radio_loadPacket(uint8_t* packet, uint16_t len);

```

Comment:
@MrD2, that a good suggestion, but we really want to keep control over the width of the parameters. The change from `uint8_t` to `uint16_t` isn't because that platform uses some different micron-controller, it's because we are now adding support for IEEE802.15.4g, in which link-layer frames can be much longer (max 2047 bytes IIRC) compared to IEEE802.15.4 (127 bytes). So I suggest so still use the explicit `uint16_t` signature